src/core/operations/Murmurhash3.mjs

```diff
@@ -0,0 +1,113 @@
+/**
```

Comment:
This file seems to verbatim copy the MIT Murmur implementation by GaryCourt, so should likely include their license information.  See an example here: https://github.com/gchq/CyberChef/blob/df151eabf91cdb31292cfa336844bfdc24afba19/src/core/operations/ToQuotedPrintable.mjs#L4